baselines/common/cmd_util.py

```diff
@@ -16,30 +16,58 @@
 from baselines.common.atari_wrappers import make_atari, wrap_deepmind
 from baselines.common.vec_env.subproc_vec_env import SubprocVecEnv
 from baselines.common.vec_env.dummy_vec_env import DummyVecEnv
-from baselines.common.retro_wrappers import RewardScaler
+from baselines.common import retro_wrappers
+import functools
```

Comment:
I think this isn't used anywhere

Reply:
yes, thanks! flake8 flags it now (was delayed due to github issues)